routers/organizationRouter.go

```diff
@@ -236,27 +219,23 @@ func (handler *organizationHandler) AddMemberToOrganization(writer http.Response
 		http.Error(writer, "No Organization ID Found", http.StatusBadRequest)
 		return
 	}
-	userId := request.FormValue("user_id")
-	if userId == "" {
+	newMemberId := request.FormValue("user_id")
```

Comment:
Personally think the interchangeable language of member vs user should be standardized to one or the other (my work has used user as that is the actual datatype) (this is nitpicking and can be ignored)

Reply:
In other methods `userId` was talking about the user who sent the request. Using userId as the name of the new user/member in an org could be a bit confusing. Also, being that the route has `/members` it seemed fitting in this route. But, I'm down to change it. We should make a ticket for this to standardize.

Reply:
i also think /members is a bit wack too but w/e
while we could make a ticket for this, it's defo a strong contender for most pointless backlog ticket